tests/rabbit/mod.rs

```diff
@@ -0,0 +1,93 @@
+use chrono::Utc;
+use futures::TryStreamExt;
+use lapin::options::{BasicAckOptions, BasicConsumeOptions, QueueBindOptions, QueueDeclareOptions};
+use lapin::types::FieldTable;
+use lapin::{Connection, ConnectionProperties, Consumer, ExchangeKind};
+use uuid::Uuid;
+
+use esrs::event_bus::rabbit::{RabbitEventBus, RabbitEventBusConfig};
+use esrs::event_bus::EventBus;
+use esrs::StoreEvent;
+
+use crate::aggregate::{TestAggregate, TestEvent};
+
+#[tokio::test]
+async fn rabbit_event_bus_test() {
+    let rabbit_url: String = std::env::var("RABBIT_URL").unwrap();
+
+    let exchange: &str = "test";
+    let queue: &str = "queue";
+    let routing_key: &str = "test_routing_key";
+
+    let config: RabbitEventBusConfig = RabbitEventBusConfig::builder()
+        .url(rabbit_url.as_str())
+        .exchange(exchange)
+        .exchange_kind(ExchangeKind::Fanout)
+        // .publish_routing_key(Some(routing_key.to_string()))
```

Comment:
```suggestion
```

Reply:
Should this be removed?

Reply:
Mmmh i think we can keep it. Let me uncomment that